### PR TITLE
NOISSUE - Rename MF_ to MG_

### PR DESCRIPTION
**Pull request by JeffMboya**
### What does this do?
This PR renames all instances of "MF_" to "MG_" .

### Which issue(s) does this PR fix/relate to?
Resolves #180

### List any changes that modify/break current functionality
This PR might cause issues if there are dependencies or references that still use the old "MF_" prefix. All such dependencies and references should be updated to use the new "MG_" prefix.

### Have you included tests for your changes?
No. 

### Did you document any new/modified functionality?
No new functionality was introduced.

### CODE REVIEW

File: docs/tracing.md [hunk 1]
```diff
@@ -28,11 +28,11 @@ Jaeger uses 5 ports within the Magistrala framework. These ports can be edited i
 | MF_JAEGER_FRONTEND  | UI port                                           | 16686       |
 | MF_JAEGER_COLLECTOR | Collector for jaeger.thrift directly from clients | 14268       |
 | MF_JAEGER_CONFIGS   | Configuration server                              | 5778        |
-| MF_JAEGER_URL       | Jaeger access from within Magistrala                | jaeger:6831 |
+| MF_JAEGER_URL       | Jaeger access from within Magistrala              | jaeger:6831 |
```
Comment (arvindh123) [inline docs/tracing.md]: Please change `MF_` to `MG_` in all places

File: docs/benchmark.md [hunk 2]
```diff
@@ -135,6 +135,6 @@ TBD
 [satori]: https://github.com/satori-com/mzbench
 [mzbench-cli]: https://github.com/mzbench/mzbench/blob/master/doc/cli.md
 [mzbench-dashboard]: https://github.com/mzbench/mzbench/blob/master/doc/dashboard.md
-[mf-benchmark]: https://github.com/mainflux/benchmark/terr/main/mzbench
+[mf-benchmark]: https://github.com/mainflux/benchmark/tree/master/mzbench
```
Comment (arvindh123) [inline docs/benchmark.md]: I think, This also need to move to `absmach`

File: docs/kubernetes.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ helm install ingress-nginx ingress-nginx/ingress-nginx --version 3.26.0 --create
 Get Helm charts from [Magistrala DevOps GitHub repository][devops-repo]:
 
 ```bash
-git clone https://github.com/mainflux/devops.git
+git clone https://github.com/absmach/devops.git
 cd devops/charts/mainflux
```
Comment (arvindh123) [inline docs/kubernetes.md]: We need to rename `mainflux` to `magistrala` HELM chart repo

Comment (arvindh123) [reply docs/kubernetes.md]: Lets create new issue for that in both Magistrala repo and HELM chart repo.

File: docs/kubernetes.md [hunk 4]
```diff
@@ -232,24 +232,24 @@ You can check if they are succesfully created:
...
 ```
 
-[devops-repo]: https://github.com/mainflux/devops
+[devops-repo]: https://github.com/absmach/devops
 [kubernetes-setup]: https://kubernetes.io/docs/setup/
 [kubectl-setup]: https://kubernetes.io/docs/tasks/tools/install-kubectl/
 [helm-setup]: https://helm.sh/docs/intro/install/
 [nginx-ingress]: https://kubernetes.github.io/ingress-nginx/deploy/
-[ingress-yaml]: https://github.com/mainflux/devops/blob/master/charts/mainflux/templates/ingress.yaml#L141
+[ingress-yaml]: https://github.com/absmach/devops/blob/master/charts/mainflux/templates/ingress.yaml#L141
```
Comment (arvindh123) [inline docs/kubernetes.md]: We need to rename mainflux to magistrala HELM chart repo

File: docs/kubernetes.md [hunk 5]
```diff
@@ -232,24 +232,24 @@ You can check if they are succesfully created:
...
 [ingress-controller-args]: https://kubernetes.github.io/ingress-nginx/user-guide/cli-arguments/
 [ingress-controller-tcp-udp]: https://kubernetes.github.io/ingress-nginx/user-guide/exposing-tcp-udp-services/
 [authentication]: /authentication
 [makefile]: https://github.com/absmach/magistrala/blob/master/docker/ssl/Makefile
-[secrets]: https://github.com/mainflux/devops/blob/master/charts/mainflux/secrets/secrets.sh
+[secrets]: https://github.com/absmach/devops/blob/master/charts/mainflux/secrets/secrets.sh
```
Comment (arvindh123) [inline docs/kubernetes.md]: We need to rename mainflux to magistrala HELM chart repo

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 # Contributing to Magistrala
 
 The following is a set of guidelines to contribute to Magistrala and its libraries, which are
-hosted on the [Magistrala Organization](https://github.com/mainflux) on GitHub.
+hosted on the [Magistrala Organization](https://github.com/absmach/magistrala) on GitHub.
 
 This project adheres to the [Contributor Covenant 1.2](http://contributor-covenant.org/version/1/2/0).
 By participating, you are expected to uphold this code. Please report unacceptable behavior to
```
Comment (arvindh123) [inline CONTRIBUTING.md]: @dborovcanin Do we need to change the email domain to Magistrala ?

File: docs/dev-guide.md [hunk 7]
```diff
@@ -297,24 +297,24 @@ Please assure that MQTT microservice has `node_modules` installed, as explained
...
 [go-install]: https://golang.org/doc/install
 [mf-ci-scripts]: https://github.com/absmach/magistrala/blob/master/scripts/ci.sh
 [scratch-docker]: https://hub.docker.com/_/scratch/
-[cleanup-docker]: #cleaning-up-your-dockerized-mainflux-setup
+[cleanup-docker]: #cleaning-up-your-dockerized-magistrala-setup
 [docker-compose-ref]: https://docs.docker.com/compose/reference/overview/
 [docker-compose-extend]: https://docs.docker.com/compose/extends/
 [increase-nodes-memory]: https://medium.com/tomincode/increasing-nodes-memory-337dfb1a60dd
 [mf-aedes]: https://github.com/absmach/magistrala/blob/master/docker/aedes.yml#L31
 [aedes-mqtt-broker]: https://github.com/mcollina/aedes
```
Comment (arvindh123) [inline docs/dev-guide.md]: This should be `mg-aedes`, Please check with respective aedes.yml and change accordingly

File: docs/dev-guide.md [hunk 8]
```diff
@@ -297,24 +297,24 @@ Please assure that MQTT microservice has `node_modules` installed, as explained
...
-[mainflux-repo]: https://github.com/Magistrala/mainflux
+[magistrala-repo]: https://github.com/absmach/magistrala
 [protocol-buffers]: https://developers.google.com/protocol-buffers/
 [golang-protobuf]: https://github.com/golang/protobuf
 [protobuf-install]: https://github.com/golang/protobuf#installation
 [protobuf]: https://github.com/google/protobuf
-[google-protobuf]: "google.golang.org/protobuf/proto"
+[google-protobuf]: https://google.golang.org/protobuf/proto
 [go-install]: https://golang.org/doc/install
 [mf-ci-scripts]: https://github.com/absmach/magistrala/blob/master/scripts/ci.sh
```
Comment (arvindh123) [inline docs/dev-guide.md]: It should be `mg-ci-scripts`

**Review by arvindh123 — COMMENTED** [review]
Please change to MG from MF in following places

https://github.com/JeffMboya/magistrala-docs/blob/3ef79fd68eb9e422263d539115ecf82bc693b75d/CONTRIBUTING.md#L56

https://github.com/JeffMboya/magistrala-docs/blob/3ef79fd68eb9e422263d539115ecf82bc693b75d/CONTRIBUTING.md#L60

https://github.com/JeffMboya/magistrala-docs/blob/3ef79fd68eb9e422263d539115ecf82bc693b75d/CONTRIBUTING.md#L83

**Comment by JeffMboya** [comment]
> Please change to MG from MF in following places
> 
> https://github.com/JeffMboya/magistrala-docs/blob/3ef79fd68eb9e422263d539115ecf82bc693b75d/CONTRIBUTING.md#L56
> 
> https://github.com/JeffMboya/magistrala-docs/blob/3ef79fd68eb9e422263d539115ecf82bc693b75d/CONTRIBUTING.md#L60
> 
> https://github.com/JeffMboya/magistrala-docs/blob/3ef79fd68eb9e422263d539115ecf82bc693b75d/CONTRIBUTING.md#L83

Done.

**Comment by arvindh123** [comment]
@JeffMboya , Please resolve conflict and rebase branch to `main`

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**